Libs/Reflex/src/main/java/reflex/node/IdentifierNode.java

```diff
@@ -51,11 +54,17 @@ public ReflexValue evaluate(IReflexDebugger debugger, Scope scope) {
         // a resolution. If we resolve to something and that variable type is a map, use the remaining
         // part to resolve to a value using map lookup semantics.
 
-        ReflexValue value = scope.resolve(identifier, namespacePrefix);
-        if (value == null) {
-            value = resolveMapDotter(scope);
+        ReflexValue value;
```

Comment:
Checked in by mistake. Was related to syntax checking - that approach was abandoned.